Taskfile.yml

```diff
@@ -11,11 +11,44 @@ tasks:
     desc: Initialize the backend project
     dir: tdrs-backend
     cmds:
+      - task: create-network
+      - task: install-sentry
       - docker-compose -f docker-compose.yml up -d --build
       - docker-compose -f docker-compose.yml exec web sh -c "python ./manage.py makemigrations"
       - docker-compose -f docker-compose.yml exec web sh -c "python ./manage.py migrate"
       - docker-compose -f docker-compose.yml down
 
+  clone-sentry-repo:
+    desc: Clone the sentry repo
+    dir: sentry
+    cmds:
+      - git clone https://github.com/getsentry/self-hosted.git || true
+
+
+  test-sentry:
+    desc: Run sentry tests
+    dir: sentry
+    cmds:
+      # limiting the memory to 2GB and CPU to only one cpu @0, for faster response, you can remove the limittask :  --cpuset-cpus 0 
+      - (docker run --privileged -p 9000:9000 -d --memory="6g" --memory-swap="20g" --name sentry docker:dind) || true
```

Comment:
The minimum memory we could set is 6g, anything below 6G, linux kernel will shutdown the instance